Update of SCIP version

Not important now, but I'm just wondering how important it might be to update the version of SCIP interface used in DSP. The version of SCIP currently required by DSP if I understand correctly is currently 3.1.1, whereas the most recent version of SCIP is 7.0.0.

When I ran `make` for compiling DSP, I got the following interface related error:

```
[ 85%] Building CXX object src/CMakeFiles/DspObj.dir/SolverInterface/OsiScipSolverInterface.cpp.o
/home/bdandurand/DSP/src/SolverInterface/OsiScipSolverInterface.cpp:292:18: error: no matching function for call to 'SCIPtrySolFree'
        SCIP_CALL_ABORT(SCIPtrySolFree(scip_, &sol, true, true, true, true, &stored));
                        ^~~~~~~~~~~~~~
/home/bdandurand/scipoptsuite-7.0.0/scip/src/scip/def.h:340:44: note: expanded from macro 'SCIP_CALL_ABORT'
                          if( (_restat_ = (x)) != SCIP_OKAY )                                                 \
                                           ^
/home/bdandurand/scipoptsuite-7.0.0/scip/src/scip/scip_sol.h:1278:14: note: candidate function not viable: requires 8 arguments, but 7 were provided
SCIP_RETCODE SCIPtrySolFree(
             ^
1 error generated.
```

--- a/src/SolverInterface/OsiScipSolverInterface.cpp
+++ b/src/SolverInterface/OsiScipSolverInterface.cpp
@@ -286,10 +286,10 @@ void OsiScipSolverInterface::setColSolution(const double* colsol) {
 	SCIP_CALL_ABORT(SCIPsetSolVals(scip_, sol, SCIPgetNOrigVars(scip_), &vars_[0], &solution_[0]));
 
 	/** check solution and free if infeasible */
-#if SCIP_VERSION_MAJOR >= 6
-	SCIP_CALL_ABORT(SCIPtrySolFree(scip_, &sol, true, true, true, true, true, &stored));
-#else
+#if SCIP_VERSION_MAJOR < 6
 	SCIP_CALL_ABORT(SCIPtrySolFree(scip_, &sol, true, true, true, true, &stored));
+#else
+	SCIP_CALL_ABORT(SCIPtrySolFree(scip_, &sol, true, true, true, true, true, &stored));
 #endif
 }
 

